test/NCI.OCPL.Api.BestBets.Tests/Tests/Models/CancerGovBestBetsSynonymTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+using Xunit;
+
+using NCI.OCPL.Utils.Testing;
+
+using NCI.OCPL.Api.BestBets.Tests.CategoryTestData;
+using System.Xml;
+using System.IO;
+using System;
+
+namespace NCI.OCPL.Api.BestBets.Tests
+{
+    /// <summary>
+    /// Tests for the CancerGovBestBet class, 
+    /// and by extension the CancerGovBestBetSynonym class.
+    /// These are really just deserialization tests,    
+    /// </summary>
+    public class CancerGovBestBetSynonymTest
+    {
+        [Fact]
+        public void GetSchema_ReturnsNull()
+        {
+            CancerGovBestBetSynonym bestBetSyn = new CancerGovBestBetSynonym();
+
+            Assert.Null(bestBetSyn.GetSchema());
+        }
+
+        [Fact]
+        public void WriteXML_ThrowsNotSupportedException()
+        {
+            CancerGovBestBetSynonym bestBetSyn = new CancerGovBestBetSynonym();
+            XmlWriterSettings settings = new XmlWriterSettings()
+            {
+                Indent = true,
+                IndentChars = "\t"
+            };
+
+
+            string TestFilePath = "CGBBCategory.BreastCancer.xml";
+            MemoryStream stream = new MemoryStream();
+            StreamWriter writer = new StreamWriter(stream);
+            XmlWriter xmlWriter = XmlWriter.Create(writer);
```

Comment:
Technically, instead of attempting to write a file, you could use a StringWriter().  We don't really want a file created, we just want to know that an exception is thrown.